src/lib/button/button.md

```diff
@@ -8,23 +8,22 @@ and accessible experience for users. A `<button>` element should be used wheneve
 is performed. An `<a>` element should be used whenever the user will _navigate_ to another view.
 
 
-There are five button variants, each applied as an attribute:
+There are seven button variants, each applied as an attribute:
```

Comment:
Can we change this to "several button variants"? Otherwise we'll keep forgetting to update the number when we decide to add more.